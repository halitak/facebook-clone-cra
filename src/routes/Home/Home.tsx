import React, { FunctionComponent } from 'react';
import { Link } from 'react-router-dom';
import Sidenav from '../../components/Sidenav';

const Home: FunctionComponent = () => (
  <div>
    <Sidenav />
    <h1>Home page</h1>
    <Link to="/feed">Go to feed</Link>
  </div>
);

export default Home;
