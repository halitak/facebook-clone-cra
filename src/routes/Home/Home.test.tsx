import React from 'react';
import { shallow } from 'enzyme';
import Home from './Home';

it('renders corretly', () => {
  const wrapper = shallow(<Home />);
  expect(wrapper).toMatchSnapshot();
});
