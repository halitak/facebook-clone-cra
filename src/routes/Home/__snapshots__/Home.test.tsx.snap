// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders corretly 1`] = `
<div>
  <h1>
    Home page
  </h1>
  <Button
    primary={true}
  >
    Primary Button
  </Button>
  <Button
    secondary={true}
  >
    Secondary Button
  </Button>
  <Link
    to="/feed"
  >
    Go to feed
  </Link>
</div>
`;
